### PR TITLE
Bugfix/copy changes

### PR DESCRIPTION
**Pull request by johnnieskywalker**
# [Fixed issue](https://www.notion.so/oazo/Multiply-MVP-tests-Feedback-bugs-2a776db8d0cf4acdb42717c96ccdf9d2?d=4427d71659d847c6899daf9023925dd0#0e123d4865934b6da4905f4a8fe11485)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
Fixed following bug 
Borrow up to 1000 Dai for every 100,000 of collateral". This doesn't make much sense and not every collateral has the same ratio.  
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

**Comment by johnnieskywalker** [comment]
fix:
image

**Comment by johnnieskywalker** [comment]
Made maxBorrow DAI an integer 
image

File: features/openVaultOverview/useRedirectToOpenVault.tsx [hunk 1]
```diff
@@ -15,7 +15,6 @@ export function useRedirectToOpenVault() {
         ilk: ilk,
         token: token,
         liquidationRatio: parseFloat(liquidationRatio.toFixed(2)),
```
Comment (pacxiu) [inline features/openVaultOverview/useRedirectToOpenVault.tsx:17]: This should be BigNumber

File: features/openVaultOverview/SelectVaultTypeModal.tsx [hunk 2]
```diff
@@ -11,20 +10,13 @@ interface Props {
...
-  close,
-  balance,
-}: ModalProps<Props>) {
+export function SelectVaultTypeModal({ ilk, token, liquidationRatio, close }: ModalProps<Props>) {
   const { t } = useTranslation()
 
   const exposureMultiplier = (1 + 1 / (liquidationRatio - 1.0)).toFixed(2)
 
-  const maxBorrowAmount = ((1 / liquidationRatio) * 100000).toFixed(2)
+  const maxBorrowAmount = ((1 / liquidationRatio) * 100000).toFixed(0)
```
Comment (pacxiu) [inline features/openVaultOverview/SelectVaultTypeModal.tsx:19]: This should use BigNumbers for calcs. You can later on use `formatAmount` or `formatCryptoBalance` to display it on FE

**Review by pacxiu — CHANGES_REQUESTED** [review]

File: features/openVaultOverview/SelectVaultTypeModal.tsx (hunk 2 above)
Comment (johnnieskywalker) [reply features/openVaultOverview/SelectVaultTypeModal.tsx:19]: copy that ser changed this way:
```
  const exposureMultiplier = one.plus(one.div(liquidationRatio.minus(one))).toFixed(2)
  
  const maxBorrowAmount = new BigNumber(one.div(liquidationRatio).multipliedBy(100000)).toFixed(0)
```

**Comment by johnnieskywalker** [comment]
Screenshot taken after code review changes, doing calculations on BigNumbers now 
image

**Review by pacxiu — APPROVED** [review]

**Pull request merged**